src/Aspire.Dashboard/Aspire.Dashboard.csproj

```diff
@@ -3,10 +3,9 @@
   <PropertyGroup>
     <!-- This list of runtimes needs to match the set of runtimes specified in eng/workloads/workloads.csproj -->
     <RuntimeIdentifiers>win-x86;win-x64;win-arm64;linux-x64;linux-arm64;osx-x64;osx-arm64</RuntimeIdentifiers>
-    <OutputType>Exe</OutputType>
     <TargetFramework>$(NetCurrent)</TargetFramework>
-    <Nullable>enable</Nullable>
-    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Why were these removed?

Reply:
They are unnecessary since these are the default values.

* OutputType gets defaulted to Exe by `<Project Sdk="Microsoft.NET.Sdk.Web">`
* Nullable and ImplicitUsings gets defaulted to enable in [Directory.Build.props](https://github.com/dotnet/aspire/blob/0d74a1466004eceb60ce02fbf369e72d9c399b91/Directory.Build.props#L24-L25).